app/src/main/java/app/revanced/manager/domain/manager/base/BasePreferencesManager.kt

```diff
@@ -64,6 +64,7 @@ abstract class Preference<T>(
     val flow = dataStore.data.map { with(it) { read() } ?: default }.distinctUntilChanged()
 
     suspend fun get() = flow.first()
+    fun defaultValue() = default
```

Comment:
This was named defaultValue because of this:
```
app/src/main/java/app/revanced/manager/domain/manager/base/BasePreferencesManager.kt:59:15 Platform declaration clash: The following declarations have the same JVM signature (getDefault()Ljava/lang/Object;):
    fun <get-default>(): T defined in app.revanced.manager.domain.manager.base.Preference
    fun getDefault(): T defined in app.revanced.manager.domain.manager.base.Preference
```

Reply:
`default` is declared as a `protected val` in the `Preference` constructor. You can make it public to get rid of the function